packages/core/src/http-client/http-client.ts

```diff
@@ -262,14 +262,12 @@ export function shouldHandleCsrfToken(
   );
 }
 
-export const xCsrfTokenHeaderKey = 'x-csrf-token';
```

Comment:
Although I think this really wasn't necessary, technically this is a breaking change...
Consider just deprecating it (and turning off the linter for this).
However, I am not really worried that this was used by anyone ;)
So we could also just mention this in the changelog as wrongfully exposed variable or so...

Reply:
This line was added recently and was not released. Worst case is, this PR misses the next release and people may start using it. I would take this risk as breaking changes;)